### PR TITLE
Use Namedreals for system mines in pedia

### PR DESCRIPTION
**Pull request by MichaelCourtney**

### CODE REVIEW

**Comment by MichaelCourtney** [comment]
Pedia update for release

**Comment by agrrr3** [comment]
LGTM (naming of the named values could be probably more signifying, but OK)

File: default/stringtables/en.txt [hunk 1]
```diff
@@ -13968,21 +13968,21 @@ System Defense Mines 1
...
+The [[metertype METER_STRUCTURE]] of all ships belonging to an enemy empire which are present in a system containing a planet owned by the empire that possesses this technology will be reduced at a rate of [[value SYSTEM_MINES_1]] per turn.
 
 Mines regenerate each turn, so are not depleted by use.'''
 
 DEF_SYST_DEF_MINE_2
 System Defense Mines 2
 
 DEF_SYST_DEF_MINE_2_DESC
-Reduces the [[metertype METER_STRUCTURE]] of enemy ships by an additional 32 per turn, cumulative with [[tech DEF_SYST_DEF_MINE_1]].
+Reduces the [[metertype METER_STRUCTURE]] of enemy ships a total of [[value SYSTEM_MINES_2]] per turn.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: you changed the description from "cumulative" to "a total of" but don't appear to change the scripts. Was the text out of date before this, or is it still cumulative?

Comment (MichaelCourtney) [reply default/stringtables/en.txt]: The macros themselves are based on a priority system by my reading.

// This macro handles multiple entries, where only the earliest applicable effects a target.

The old stringtable entries were fixed values rather then in-game.

Comment (geoffthemedio) [reply default/stringtables/en.txt]: should probably make the descriptions clear about whether the effects are cumulative or not, regardless of how that's implemented...

Comment (MichaelCourtney) [reply default/stringtables/en.txt]: do you mean, "a total of, not cumulative with.." or something similar?

Comment (geoffthemedio) [reply default/stringtables/en.txt]: yes. "a total of" is probably better than "not cumulative with", though it's misleading if there can be something else that overrides it.

**Comment by MichaelCourtney** [comment]
Is this clearer?

**Comment by Vezzra** [comment]
This seems very close to be ready for merge, so I've postponed RC1 a bit. Can we merge this within the next two days? I'd like to include it into the release...

**Pull request merged**